alacritty/src/renderer/mod.rs

```diff
@@ -417,6 +417,7 @@ struct InstanceData {
     bg_g: f32,
     bg_b: f32,
     bg_a: f32,
+    colored: u8,
```

Comment:
Adding a comment above this for uniformity would probably be a good idea.

Reply:
Done, should also be more explicit with the new field name.